woocommerce/storefront
# document release process

Adds a new document with details of how to release Storefront.

### How to test the changes in this Pull Request:
1. Read the [releases.md doc](https://github.com/woocommerce/storefront/blob/add/release-process-doc/docs/releases.md). Ensure this provides good guidance for how to release this theme, and doesn't include details that we'll need to update too often.

### Changelog

> Documented Storefront release process.
